### PR TITLE
Focalboard Notice

### PR DESCRIPTION
**Pull request by chenilim**
Adding a notice for Focalboard in v5.36+.

### CODE REVIEW

File: notices.json [hunk 1]
```diff
@@ -1,5 +1,23 @@
...
+          "audience": "sysadmin",
+          "clientType": "all",
+          "serverVersion": [">5.35"],
+          "displayDate": ">= 2021-06-16T00:00:00Z"
+        },
+        "localizedMessages": {
+          "en": {
+            "title": "Enable Focalboard",
+            "description": "Focalboard integrates project management tools like kanban boards, cards, and tables with Mattermost to help you easily plan projects, manage tasks, and run effective meetings. Enable Focalboard today following the steps in the link below.",
+            "image": "https://user-images.githubusercontent.com/46905241/121941290-ee355280-cd03-11eb-9b9f-f6f524e4103e.gif",
```
Comment (chenilim) [inline notices.json:14]: Hi @esethna or @ogi-m, the image is hosted from [this discussion](https://github.com/mattermost/focalboard/discussions/578). Is there a better place to put images?

Comment (esethna) [reply notices.json:14]: Hey @chenilim, instructions for images are here: https://github.com/mattermost/notices#opening-a-pr

You can add it in this PR branch to the images folder so it gets merged into the release branch where the URL will be pointing

**Review by esethna — CHANGES_REQUESTED** [review]
Can you also help to rebase this to the `release` branch?

File: notices.json [hunk 2]
```diff
@@ -1,5 +1,23 @@
...
+            "actionText": "Learn more",
+            "actionParam": "https://www.focalboard.com/download/mattermost"
```
Comment (esethna) [inline notices.json:16]: I think this would really help lower the barrier to entry here: https://mattermost.atlassian.net/browse/MM-36441

**Review by esethna — CHANGES_REQUESTED** [review]

File: notices.json (hunk 1 above)
Comment (chenilim) [reply notices.json]: Thanks @esethna! Sorry, I missed that part. Should be corrected now - though we would need to commit this before we can double-check that the image URL is right.

File: notices.json (hunk 2 above)
Comment (chenilim) [reply notices.json:16]: This would be nice-to-have, but moot for Focalboard once we are enabled by default.

**Comment by chenilim** [comment]
> Can you also help to rebase this to the `release` branch? https://github.com/mattermost/notices#opening-a-pr

Thanks @esethna. I'll go ahead and merge this PR to main, and open another one to merge to release once we finalize the text. Thanks.

File: notices.json (hunk 2 above)
Comment (esethna) [reply notices.json:16]: Ah. If that's the case curious why the text is asking admins to follow the steps to enable focalboard? Sorry, I might be missing something here. @chenilim

Comment (chenilim) [reply notices.json:16]: The Focalboard websocket URL needs to be added to the web-proxy (nginx, Apache) config (see [thread](https://community.mattermost.com/core/pl/96d1hz9o7fdrzdfh7ettrjcbzo)). Our mitigation is:
* Show an [error banner](https://github.com/mattermost/focalboard/pull/570) with a link to the docs on websocket errors, and
* (In the next release) Add a [link to docs](https://focalboard-community.octo.mattermost.com/workspace/qgsck6cts3fwpqwyjiupjm5cde/shared?id=47aa9bb4-6967-4a96-83c7-11bd6b20f1eb&v=ca1a5441-4c1c-4d0e-856e-88cc744576c8&r=762db9c6-305c-4d13-8ce6-9f11e5232c24&c=a67e26c8-4e40-4c86-8a5a-c336f36bcf99) in the plugin config screen.

Comment (esethna) [reply notices.json:16]: Got it, thanks for the context

**Pull request merged**